layouts/_default/single.html

```diff
@@ -11,6 +11,7 @@
       </div>
       <div class="col-xs-12 col-md-8 content-container">
         <div class="content">
+          <button id="edit-on-github" data-toggle="popover" type="button" class="btn btn-default" style="float: right;" data-original-title="" title=""><i class="fa fa-github"></i>&nbsp;&nbsp;Edit On Github</button>
```

Comment:
embedding style in html is bad practice
use bootstrap's "pull-right" class instead